### DISCUSSION TITLE
[Bug] The deployment of Skywalking Showcase v8 on docker platform is broken

### DISCUSSION THREAD

**Discussion by sun-rui** [apache/skywalking#9321] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Showcase (apache/skywalking-showcase)

### What happened

Following the showcase v8 readme: https://skywalking.apache.org/docs/skywalking-showcase/v8/readme/, the deployment on docker platform failed.
```
git checkout -b v8 origin/8.x
make deploy.docker
...
Error response from daemon: manifest unknown
```

### What you expected to happen

The deployment should be successful

### How to reproduce

The reason is that the requested tag of the container images for the 'agent' feature does not exist.
The requested tag is:
```
$ git rev-parse --short HEAD
6625e5a
```
According to the several most recent commit IDs, I found the closest tag ccfe0a3 whose corresponding images exist:
```
commit 6625e5af4b9001fbd68a1c1a417f109e1c93e750 (HEAD -> v8, origin/8.x)
Author: 吴晟 Wu Sheng <wu.sheng@foxmail.com>
Date:   Mon Jun 13 22:43:32 2022 +0800

    Update readme.md

commit 0641591aa102da68a895c50de6b9c71d17c68729
Author: Wu Sheng <wu.sheng@foxmail.com>
Date:   Wed Mar 30 16:46:22 2022 +0800

    Set up 8.x branch for legacy showcase

commit ccfe0a353bdf904ddb74af3c540408f26a9e4b6a
Author: 吴晟 Wu Sheng <wu.sheng@foxmail.com>
Date:   Thu Mar 24 17:38:51 2022 +0800

    Update Java agent to fix thread pool metric (#17)
```
After I made the following change to Makefile.in, the images were successfully pulled.
```
-TAG ?= $(shell git rev-parse --short HEAD)
+TAG ?= ccfe0a3
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Which is missing exactly?

**Comment by wu-sheng** [comment]
I can find the only update is about Java agent image.

But the one could be found here, https://github.com/apache/skywalking-java/pkgs/container/skywalking-java%2Fskywalking-java/17511501?tag=8a0f79743cee6bc429218928f6afb296ed758ea4-java8

**Comment by sun-rui** [comment]
The point is I followed the readme, but I failed to depoly the showcase v8 on docker. I managed to fix it by myself, which is a trick. I reported it here so you guys can figure out an appropriate solution, either documentation update or code update.

**Comment by wu-sheng** [comment]
We don't maintain v8 anymore. So, we only would spend very little time, if there is a very urgent issue.
I compared two commit IDs, but that is the only difference I found. And be confused.

**Reply by sun-rui** [reply]
The Makefile use the ID of the latest commit as the tag to pull container images, but the images do not exist. The commits themselves are not relevant.
If showcase V8 is not maintained, maybe it is better to remove it from the official documentation. Otherwise users may be confused with a bad experience. I wanted to give showcase v8 a try instead of v9 just because I met another problem with V9, but that's another story.

**Comment by wu-sheng** [comment]
Moving to discuss

**Comment by kezhenxu94** [comment, marked as answer]
> $ git rev-parse --short HEAD
> 6625e5a

This is because we didn't publish Docker images for this commit (it is in branch `8.x` and we don't run workflow for this branch), I've added the workflow to `8.x` branch so you can find the images now (after the workflow completes) https://github.com/apache/skywalking-showcase/tree/8.x

@sun-rui just update your local codes and re-deploy, it should be OK now

**Reply by sun-rui** [reply]
thanks!
